### PR TITLE
Add tests for FS overhead methods

### PR DESCRIPTION
**Pull request by jkonecny12**
Tests for https://github.com/rhinstaller/blivet/pull/287

### CODE REVIEW

File: tests/formats_test/misc_test.py [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+from blivet.formats.fs import *
+from blivet.size import Size
+
+class FSOverheadTestCase(unittest.TestCase):
+
+    def testRequiredSizeFS(self):
+        # FS is abstract parent which doesn't have metadata
+        self.assertEqual(FS.getRequiredSize(Size("100 MiB")), Size("100 MiB"))
+        self.assertEqual(Ext2FS.getRequiredSize(Size("100 B")), Size("107 B"))
```
Comment (vpodzime) [inline tests/formats_test/misc_test.py]: I doubt an ext2 file system this big could ever be created. ;) Maybe change the unit to ``MiB``?

**Comment by vpodzime** [comment]
Looks good to me otherwise.

File: tests/formats_test/misc_test.py (hunk 1 above)
Comment (jkonecny12) [inline tests/formats_test/misc_test.py]: I'll change it locally to ``Ext2FS.getRequiredSize(Size("100 MiB")), Size("100 MiB") / 0.93)``

**Comment by jkonecny12** [comment]
Pushed

**Comment by dwlehman** [comment]
This introduced a pylint warning which prevents successful `make check`:

`W0401(wildcard-import):   3,0: : Wildcard import blivet.formats.fs`

Please fix it.

**Pull request closed without merging**